Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM golang
+
+RUN go get github.com/Financial-Times/coco-synthetic-image-publication-monitor
+RUN cp $GOPATH/bin/coco-synthetic-image-publication-monitor /synth-publication
+RUN cp $GOPATH/src/github.com/Financial-Times/coco-synthetic-image-publication-monitor/attributes.template /attributes.template
+RUN cp $GOPATH/src/github.com/Financial-Times/coco-synthetic-image-publication-monitor/systemAttributes.template /systemAttributes.template
+RUN cp $GOPATH/src/github.com/Financial-Times/coco-synthetic-image-publication-monitor/usageTickets.template /usageTickets.template
+
+
+ENV AWS_ADDRESS s3.amazonaws.com
+ENV BUCKET_ADDRESS com.ft.imagepublish.int
+EXPOSE 8080
+
+CMD cd / && /synth-publication -dynRouting=true -postHost=$POST_ADDRESS -s3Host=$BUCKET_ADDRESS.$AWS_ADDRESS
```

Comment:
In the code the template files' paths are relative (which is not that nice), and strangely the default directory of this docker image is '/go', instead of '/' .